TLM/TLM/Manager/Impl/LaneConnectionManagerData/LaneConnectionData.cs

```diff
@@ -0,0 +1,22 @@
+namespace TrafficManager.Manager.Impl.LaneConnectionManagerData {
+    internal struct LaneConnectionData {
+        public uint LaneId;
+
+        /// <summary>
+        /// for every connection, both forward and backward connection pairs are created.
+        /// for bi-directional connection both forward and backward are enabled.
+        /// for uni-directional connection only forward connection is enabled.
+        /// if there is no connection either way, then there must be no LaneConnectionData entry.
+        /// </summary>
+        public bool Enabled;
```

Comment:
Hmm, is it just to skip checking if key exits?

Reply:
I added backward disabled lane connections for uni-directional to reduce searching later on. do you remember you said my code was looping too much? this is the solution.

Reply:
The dictionary does not reduce search at all. it was just a way of reducing memory consumption.